src/olympia/addons/tasks.py

```diff
@@ -808,3 +809,27 @@ def disable_legacy_files(ids, **kw):
             for file_ in files:
                 log.info('Disabling file %d from addon %s', file_.pk, addon.pk)
                 file_.update(status=amo.STATUS_DISABLED)
+
+
+@task
+@use_primary_db
+def remove_amo_links_in_url_fields(ids, **kw):
+    """With the specified ids, remove the AMO links in the following URL fields
+    of each add-on: homepage, support_url, contribution."""
+    log.info('Deleting AMO links in URL fields %d-%d [%d].', ids[0], ids[-1],
+             len(ids))
+    addons = Addon.objects.filter(id__in=ids)
+    for addon in addons:
+        with transaction.atomic():
+            translation_ids = []
+            if addon.homepage:
+                translation_ids.append(addon.homepage.id)
```

Comment:
you can do `if addon.homepage_id:` and use `homepage_id` directly if you just need the `id`.